serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -21,8 +21,22 @@ servershell.update_attribute = function(object_id, attribute_id, new_value, mult
     let server = servershell.get_object(object_id);
 
     // Cast new values to corresponding data type if wanted
-    new_value = attribute.type === 'number' ? Number.parseInt(new_value) : new_value;
-    new_value = attribute.type === 'boolean' ? !!new_value : new_value;
+    if (attribute.type === 'number') {
+        if (attribute.multi) {
+            new_value = new_value.map(v => Number.parseInt(v));
```

Comment:
What about if `parseInt` returns `NaN`? Is this safe to handle?

Reply:
If the user enter 'a' into a number field and ignored the red background due to regex warning it is supposed to fail here yes.

Reply:
If I see it correctly, `Number.parseInt('a')` won't fail, it would return a `NaN` number object.
![image](https://user-images.githubusercontent.com/5676670/80602896-fe22a200-8a2f-11ea-90b7-ddb8ff9c8824.png)

Reply:
It will fail at latest on the commit to the backend. Adding another part validating here would duplicate the logic imho. Again if you type a 'a' into a field where a number is expected it should jump into your face already before saving

![Screenshot from 2020-04-29 15-47-56](https://user-images.githubusercontent.com/3993306/80603587-e992d980-8a30-11ea-9799-5901e392cf02.png)

at least after saving and before commiting:

![Screenshot from 2020-04-29 15-48-04](https://user-images.githubusercontent.com/3993306/80603576-e7c91600-8a30-11ea-9450-70b26db6a185.png)

Reply:
I can add a manual extra check here but this would then be arguable the same for boolean and all other attribute types we currently have and will add in future.

Keeping the validation work in sync between backend and frontend would be tedious. For example we have a type datetime.

Reply:
Question was if it is safe to handle. If at latest on backend side it will fail, then I would consider it safe enough.

Reply:
Agreed